src/Our.Umbraco.BulkUserAdmin/Web/UI/App_Plugins/BulkUserAdmin/js/bua.controllers.js

```diff
@@ -35,28 +39,28 @@
 	        });
 	    };
 
-	    $scope.isSelected = function(usr) {
+	    $scope.isSelected = function (usr) {
 	        return _.find($scope.selectedUsers, function (itm) {
 	            return itm.Id == usr.Id;
 	        }) !== undefined;
 	    };
 
 	    $scope.isSelectedAll = function () {
-	        return $scope.users !== undefined && _.every($scope.users.items, function(itm) {
-	            return !!_.find($scope.selectedUsers, function(itm2) {
+	        return $scope.users !== undefined && _.every($scope.users.items, function (itm) {
+	            return !!_.find($scope.selectedUsers, function (itm2) {
 	                return itm.Id == itm2.Id;
 	            });
 	        });
 	    };
 
 	    $scope.selectUser = function (usr) {
-            if (!$scope.isSelected(usr)) {
-                $scope.selectedUsers.push(usr);
-            } else {
-                $scope.selectedUsers = _.reject($scope.selectedUsers, function (itm) {
-                    return itm.Id == usr.Id;
-                });
-            }
+	        if (!$scope.isSelected(usr)) {
```

Comment:
Cool, thanks!

I've just realised that the file already had a mix of tabs/spaces... our bad, doh!  Thanks for fixing the consistency. :+1: